README.md

```diff
@@ -1,22 +1,30 @@
-## eslint-config-digitalbazaar
+# Digital Bazaar ESLint Configs _(@digitalbazaar/eslint-config)_
 
 This package provides eslint rules used by Digital Bazaar as a set of extendable shared configs.
 
-There are 6 rule sets:
-1. `eslint-config-digitalbazaar`: Base rules for both node and the browser.
-2. `eslint-config-digitalbazaar/import`: Rules for es6 imports in the browser and node.
-3. `eslint-config-digitalbazaar/jsdoc`: Rules for JSDoc for both node and the browser.
-4. `eslint-config-digitalbazaar/module`: Rules for modules for both node and the browser.
-5. `eslint-config-digitalbazaar/vue`: Rules for Vue 2 projects and browser only.
-6. `eslint-config-digitalbazaar/vue3`: Rules for Vue 3 projects and browser only.
+There are various core rule sets:
+- `@digitalbazaar/eslint-config`: Base rules for both node and browser code.
+- `@digitalbazaar/eslint-config/import`: Rules for es6 imports in the browser and node code.
+- `@digitalbazaar/eslint-config/jsdoc`: Rules for JSDoc for both node and browser code.
+- `@digitalbazaar/eslint-config/module`: Rules for modules for both node and browser code.
+- `@digitalbazaar/eslint-config/vue3`: Rules for Vue 3 code.
 
+For common use cases, there are recommended configs, that load the core rules:
+- `@digitalbazaar/eslint-config/browser-recommended`: Recommended rules for projects targeting web browsers.
+- `@digitalbazaar/eslint-config/node-recommended`: Recommended rules for projects targeting node.
+- `@digitalbazaar/eslint-config/recommended`: Common recommended rules.
+- `@digitalbazaar/eslint-config/universal-recommended`: Recommended rules for projects targeting node and web browsers.
+- `@digitalbazaar/eslint-config/vue3-recommended`: Recommended rules for projects targeting Vue 3.
 
 ## Installation
 
+Requires:
+- eslint v9
+
 If you do not have eslint installed:
 ```
```

Comment:
Could you add languages to code blocks throughout?
```suggestion
```sh
```

Reply:
Added